lib/stacks/iam.ts

```diff
@@ -385,16 +377,34 @@ export class IAMStack extends Stack {
 
         const notebookPolicy = new ManagedPolicy(this, 'mlspace-notebook-policy', {
             statements: notebookPolicyStatements(this.partition, Aws.REGION),
+            description: 'Enables general MLSpace actions in notebooks and across the entire application.'
         });
 
+        // The policy that will allow for disabling services by denying IAM permissions at an application level
+        const appDenyServicesPolicy = new ManagedPolicy(this, 'mlspace-app-deny-services-policy', {
+            /**
+             * Policies must have at least one statement with an effect, actions, and resources property for validation
+             * This statement has no effect since the action is invalid
+             */
+            statements: [new PolicyStatement({
+                effect: Effect.DENY,
+                actions: ['x:*'],
```

Comment:
Does this actually deploy as is? Trying to create a policy in the console with `actions: ['x:*']` throws a validation error. Does this need to exist? Can you add some comments here explaining what this is and why it won't cause issues/if it will show up as invalid in the console explain why that's ok?